ush/python/pygfs/task/archive.py

```diff
@@ -439,7 +439,6 @@ def replace_string_from_to_file(filename_in, filename_out, search_str, replace_s
 
         return
 
-    @staticmethod
     @logit(logger)
     def _archive_expdir(arch_dict: Dict[str, Any]) -> bool:
```

Comment:
You removed the decorator, but did not actually make this an instance method by adding `self`. Also, once this is a instance method, you should no longer need to pass in a dict, since the object should already have the information contained within the instance.
```suggestion
    def _archive_expdir(self) -> bool:
```

Reply:
Thanks for catching my slop.  I added `self`, but opted to keep the `arch_dict` as the `task_config` contains all environmental variables and the configuration/execution methods should use the more limitedly scoped dictionaries (i.e. `arch_dict`).  Tested on WCOSS2 with `HPSSARCH=YES`.

Reply:
Fine for now, we can revisit later.